lib/import/BpmnImporter.js

```diff
@@ -108,7 +108,16 @@ BpmnImporter.prototype.add = function(semantic, di, parentElement) {
     // add a virtual element (not being drawn)
     element = this._elementFactory.createRoot(elementData(semantic, di));
 
-    this._canvas.setRootElement(element);
+    if (is(semantic, 'bpmn:SubProcess')) {
+      element.id = element.id + '_plane';
+    }
+
+    var plane = this._canvas.getPlane(semantic.id);
+    if (plane) {
+      this._canvas.setRootElementForPlane(element, plane);
+    } else {
+      this._canvas.createPlane(semantic.id, element);
+    }
```

Comment:
This got redundant with https://github.com/bpmn-io/diagram-js/pull/569, I will remove it